gui/ros_gui/gui/configpanel/configpanel.py

```diff
@@ -0,0 +1,357 @@
+                                                    ui.button('Choose new file:', on_click=lambda node=node, param_name=param_name: self.pick_files(node, param_name), icon='📂')
+                                            case "byte_array":
+                                                ui.input(param_text).bind_value(self.parameters, param_name).on('keydown.enter', lambda result, node=node, param_name=param_name: self.set_parameter(node, param_name, result.sender.value))
+                                            case "bool_array":
+                                                ui.input(param_text).bind_value(self.parameters, param_name).on('keydown.enter', lambda result, node=node, param_name=param_name: self.set_parameter(node, param_name, result.sender.value))
+                                            case "int_array":
+                                                ui.input(param_text).bind_value(self.parameters, param_name).on('keydown.enter', lambda result, node=node, param_name=param_name: self.set_parameter(node, param_name, result.sender.value))
+                                            case "double_array":
+                                                ui.input(param_text).bind_value(self.parameters, param_name).on('keydown.enter', lambda result, node=node, param_name=param_name: self.set_parameter(node, param_name, result.sender.value))
+                                            case "string_array":
+                                                ui.input(param_text).bind_value(self.parameters, param_name).on('keydown.enter', lambda result, node=node, param_name=param_name: self.set_parameter(node, param_name, result.sender.value))
+                                            case _:
+                                                ui.label(f"Unsupported type {param_type}").classes('text-red-500')
+                                        ui.tooltip(param_description)
+
+    def init_clients(self, active_node_list) -> dict:
+        """ Initializes the get and set parameter clients for all active nodes.
+
+        Args:
+            active_node_list (list): List of active nodes.
+
+        Returns:
+            client_list (dict): Dictionary of clients for all active nodes.
+        """
+        client_list = {}
+        for module in active_node_list:
+            get_params_client = self.create_client(ListParameters, f'/atos/{module}/list_parameters')
+            get_param_value_client = self.create_client(GetParameters, f'/atos/{module}/get_parameters')
+            set_params_client = self.create_client(SetParametersAtomically, f'/atos/{module}/set_parameters_atomically')
+            client_list[module] = {"get_params_client": get_params_client, "get_param_value_client": get_param_value_client, "set_params_client": set_params_client}
+        return client_list
+
+    def get_parameters_list(self, client_list) -> None:
```

Comment:
```suggestion
    def set_parameters_list(self, client_list) -> None:
```

Reply:
Agree, fetch is probably the most accurate. Fixed it in new commit